agent/api/container.go

```diff
@@ -78,3 +194,12 @@ func (c *Container) SetSentStatus(status ContainerStatus) {
 
 	c.SentStatus = status
 }
+
+// String returns a human readable string representation of this object
+func (c *Container) String() string {
+	ret := fmt.Sprintf("%s(%s) (%s->%s)", c.Name, c.Image, c.GetKnownStatus().String(), c.GetDesiredStatus().String())
```

Comment:
Now is probably not the right time to do this...but how do you feel about changing this format at some point in the future?